### PR TITLE
parse the right model

### PR DESCRIPTION
**Pull request by ajavanma**
parser sets the wrong model

### CODE REVIEW

**Comment by abanchev** [comment]
I think you got your logic confused. Setting GPT3.5 only sets the smart one to be 3.5 as well, while GPT4 only sets the fast one to be using 4 as well.

File: scripts/token_counter.py
Comment (nponeccop) [inline scripts/token_counter.py:1]: This is unrelated

**Review by nponeccop — CHANGES_REQUESTED** [review]
